src/exposurelog/utils.py

```diff
@@ -0,0 +1,11 @@
+__all__ = ["current_date_and_day_obs"]
+
+import astropy.time
+import astropy.units
+
+
+def current_date_and_day_obs() -> tuple[astropy.time.Time, int]:
+    """Return the current date and day_obs."""
+    current_date = astropy.time.Time.now()
+    day_obs_full = current_date.tai - 12 * astropy.units.hr
```

Comment:
Or put the hour offset into a `TimeDelta` object and pull it out as a constant so it doesn't have to be recreated each time?

Reply:
OK